sdk/dapp-kit/src/components/AccountDropdownMenu.tsx

```diff
@@ -23,6 +24,7 @@ type AccountDropdownMenuProps = {
 export function AccountDropdownMenu({ currentAccount }: AccountDropdownMenuProps) {
 	const { mutate: disconnectWallet } = useDisconnectWallet();
 	const { mutate: switchAccount } = useSwitchAccount();
+	const { data: domain } = useResolveSuiNSName(currentAccount.address);
```

Comment:
it might save some API calls to only enable this query when the current account doesn't have a label, but eh that's probably a very minor optimization

also - do we need to disable this query on any networks, e.g. localnet?

Reply:
good call on disabling.  Updated that.

It just won't find stuff on other networks which is fine I think